src/bundle/Resources/public/js/scripts/core/line.chart.js

```diff
@@ -0,0 +1,80 @@
+(function(global, doc, eZ) {
+    const MAX_NUMBER_OF_LABELS = 16;
+    const lineDefaultOptions = {
+        elements: {
+            point: {
+                radius: 2,
+            },
+            line: {
+                tension: 0,
+            },
+        },
+        scales: {
+            xAxes: [
+                {
+                    display: true,
+                    gridLines: {
+                        display: false,
+                    },
+                    ticks: {
+                        maxRotation: 0,
+                        autoSkip: false,
+                        callback: (value, index, labels) => {
+                            let labelsInterval = 1;
+
+                            if (labels.length) {
+                                labelsInterval = Math.ceil(labels.length / MAX_NUMBER_OF_LABELS);
+                            }
```

Comment:
I took me a while to understand this calculations and I think it would be more expressive when written with `Math.max`:
```suggestion
                            const labelsInterval = Math.max(Math.ceil(labels.length / MAX_NUMBER_OF_LABELS), 1);
```